File: eRxWeb/ServerModel/SelectMedicationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eRxWeb.ServerModel
{
    public class SelectMedicationModel
    {
        public SelectMedicationReturnAction ReturnAction { get; set; }
        public ScriptPadModel ScriptPadModel { get; set; }
        public string RedirectUrl { get; set; }
        public MessageModel MessageModel { get; set; }
    }
}

File: eRxWeb/SPA/Model/select-medication/select-medication.model.ts
import Scriptpadmodel1 = require("../right-panel/script-pad.model");
import ScriptPadModel = Scriptpadmodel1.ScriptPadModel;
import Messagemodel1 = require("../message.model");
import MessageModel = Messagemodel1.MessageModel;
import Constants = require("../../tools/constants");
import SelectMedicationReturnAction = Constants.SelectMedicationReturnAction;

export class SelectMedicationModel {
    public ReturnAction: SelectMedicationReturnAction;
    public ScriptPadModel: ScriptPadModel;
    public RedirectUrl: string;
    public MessageModel: MessageModel;
}